.github/optimize/scripts/healthStatusReport/createHealthStatusConfig.ts

```diff
@@ -5,32 +5,27 @@
  * except in compliance with the proprietary license.
  */
 
-import path from 'path';
-
-import { createJsonFile, isRegExp, readJsonFIle } from './services';
-import { Config } from './types';
-import { GitHubService } from './GitHubService';
+import {createJsonFile} from './services';
+import {Config} from './types';
+import {GitHubService} from './GitHubService';
 
 const ARGOCD_PROJECTS = ['optimize-previews'];
 const ARGOCD_URL = 'https://argocd.int.camunda.com';
 
 const GITHUB_TOKEN = process.env.GITHUB_TOKEN;
 const GITHUB_ORG = 'camunda';
-const GITHUB_REPO = 'camunda-optimize';
-
-const RENOVATE_CONFIG_PATH = path.join(process.cwd(), '../../renovate.json');
+const GITHUB_REPO = 'camunda';
 
 const CONFIG_FILE_NAME = 'config.json';
 
 async function createHealthStatusConfig() {
   const githubService = new GitHubService(GITHUB_TOKEN, GITHUB_ORG, GITHUB_REPO);
-
-  const renovateStringBranches = getRenovateStringBranches();
-  const releaseBranches = await githubService.getBranchesWithPrefix('release');
-  const ciBranches = [...releaseBranches, ...renovateStringBranches].sort(
+  const defaultBranch = 'main';
+  const releaseBranches = await githubService.getBranchesWithPrefix('release/optimize-');
+  const stableBranches = await githubService.getBranchesWithPrefix('stable');
+  const ciBranches = [defaultBranch, ...releaseBranches, ...stableBranches].sort(
     githubService.sortBranches,
   );
-  const maintenanceBranches = ciBranches.filter((branch) => branch.includes('maintenance'));
 
   const config: Partial<Config> = {
     argoCd: {
```

Comment:
Should we also remove ArgoCD from here?

Reply:
as per this [convo](https://camunda.slack.com/archives/C04FM0DRJT0/p1723544853899339?thread_ts=1723544541.548459&cid=C04FM0DRJT0), we remove argo